src/renderer/stores/chat/chat_scheduler.ts

```diff
@@ -119,7 +119,7 @@ export class ChatStoreScheduler {
       try {
         returnValue = await effect(...args)
       } catch (err) {
-        log.error(`Error in queuedEffect ${effectName}: ${err}`)
+        log.error(`Error in queuedEffect ${effectName}:`, err)
```

Comment:
Who was blaming me for unrelated changes? :D

Reply:
it is related, I needed this to get the real error. basically I reproduced it, got `Error in queuedEffect ...` but only he text of the error without stack trace, this change pointed me to the location in the message list store file.